### ISSUE TITLE
API10

### ISSUE THREAD

**Issue by YsEmei** [Tischel/InventorySearchBar#19]
Any plans to update this to API10? 
It's one of the plugins i miss the most after the update :/

**Comment by Tischel** [comment]
Yup just been really busy and since this plugin depends on a library from someone else I always leave it for last :P
